docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,59 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.38  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** P-RMD each SHW type
+**Data Lookup:**   
+**Function Call:** 
+- **get_component_by_id**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements
+- **get_SHW_types_and_spaces**
+- **get_SHW_equipment_connected_to_use_type**  
+
+**Applicability Checks:**
+- check that the SHW type in the U_RMD has SHW loads
+
+## Applicability Checks:
+- only projects with SHW for the SHW space type in the user model are expected to have a SHW system in the user model.  If there is no SHW in the user model, we assume there is no SHW system designed, and instead rule 11-3 Applies and P_RMD matches B_RMD.
```

Comment:
Not following what is meant by "with SHW for the SHW space type". Maybe it will become cleaerer as I review this.

Reply:
I think this is referring to how we won't expect garages to have SHW but agree it can be clarified

Reply:
This is clearer now. Recommend modifying nomenclature for clarity.